internal/internal_flags.go

```diff
@@ -46,10 +46,17 @@ const (
 	SDKFlagProtocolMessageCommand = 3
 	// SDKPriorityUpdateHandling will cause update request to be handled before the main workflow method.
 	// It will also cause the SDK to immediately handle updates when a handler is registered.
+	// This flag is currently implemented, but not set to preserve rollback safety between Go SDK 1.26.x and 1.25.x.
 	SDKPriorityUpdateHandling = 4
```

Comment:
This flag mostly applies to update processing , so another option is I rework the flag to only apply to update processing which I think I can do. That would mean the can only not be rolled back if the User was using update so should be safe for the Server team to use. Thoughts @yiminc @bergundy ?